google-cloud-vision/lib/google/cloud/vision/v1.rb

```diff
@@ -17,78 +17,78 @@
-    # ```
-    #
-    # ### Next Steps
-    # - Read the [Cloud Vision API Product documentation][Product Documentation]
-    #   to learn more about the product and see How-to Guides.
-    # - View this [repository's main README](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/README.md)
-    #   to see the full list of Cloud APIs that we cover.
-    #
-    # [Product Documentation]: https://cloud.google.com/vision
-    #
-    # ## Enabling Logging
-    #
-    # To enable logging for this library, set the logger for the underlying [gRPC](https://github.com/grpc/grpc/tree/master/src/ruby) library.
-    # The logger that you set may be a Ruby stdlib [`Logger`](https://ruby-doc.org/stdlib-2.5.0/libdoc/logger/rdoc/Logger.html) as shown below,
-    # or a [`Google::Cloud::Logging::Logger`](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-logging/latest/google/cloud/logging/logger)
-    # that will write logs to [Stackdriver Logging](https://cloud.google.com/logging/). See [grpc/logconfig.rb](https://github.com/grpc/grpc/blob/master/src/ruby/lib/grpc/logconfig.rb)
-    # and the gRPC [spec_helper.rb](https://github.com/grpc/grpc/blob/master/src/ruby/spec/spec_helper.rb) for additional information.
-    #
-    # Configuring a Ruby stdlib logger:
-    #
-    # ```ruby
-    # require "logger"
-    #
-    # module MyLogger
-    #   LOGGER = Logger.new $stderr, level: Logger::WARN
-    #   def logger
-    #     LOGGER
-    #   end
-    # end
-    #
-    # # Define a gRPC module-level logger method before grpc/logconfig.rb loads.
-    # module GRPC
-    #   extend MyLogger
-    # end
-    # ```
-    #
     module Vision
+      # rubocop:disable LineLength
+
+      ##
+      # # Ruby Client for Cloud Vision API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
+      #
+      # [Cloud Vision API][Product Documentation]:
+      # Integrates Google Vision features, including image labeling, face, logo, and
+      # landmark detection, optical character recognition (OCR), and detection of
+      # explicit content, into applications.
+      # - [Product Documentation][]
+      #
+      # ## Quick Start
+      # In order to use this library, you first need to go through the following
+      # steps:
+      #
+      # 1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+      # 2. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+      # 3. [Enable the Cloud Vision API.](https://console.cloud.google.com/apis/api/vision)
+      # 4. [Setup Authentication.](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud/master/guides/authentication)
+      #
+      # ### Preview
+      # #### ImageAnnotatorClient
+      # ```rb
+      # require "google/cloud/vision"
+      #
+      # image_annotator_client = Google::Cloud::Vision::V1.new(version: :v1)
```

Comment:
Argh... Can you manually fix this, and open an issue on gapic-generator? (It's supposed to be `Google::Cloud::Vision.new(version: :v1)`.)

Reply:
Ohh... you know what, we can't make this change yet because this isn't gapic-only....